tests/assets/spa/external-scripts/script-with-ajax.html

```diff
@@ -22,7 +22,7 @@
     var args = [function(api) {
       spaApi = api
     }]
-    scopedEE.buffer([type])
+    scopedEE.buffer([type], 'spa')
```

Comment:
because events are now in "named" buffers, we also need to make sure tests that directly modify the buffers do this too